### ISSUE TITLE
ARASAAC image filter issue

### ISSUE THREAD

**Issue by ms-mialingvo** [asterics/AsTeRICS-Grid#458]
Some of the ARASAAC images are tagged "by [ARASAAC - CC (BY-NC-SA)](https://arasaac.org/terms-of-use)" and some are tagged "by [Sergio Palao](http://www.catedu.es/arasaac/condiciones_uso.php)". The result of this dinstinction is that the 'by Sergio Palao' images don't show up when filtering for ARASAAC images. But If I'm understanding it correctly both are ARASAAC?
<img width="740" alt="image" src="https://github.com/user-attachments/assets/9cb13962-4551-4f9b-9ca9-deed07851aa9">
<img width="757" alt="image" src="https://github.com/user-attachments/assets/b2d15554-1e0b-45d7-ba4e-517f5858fce0">

**Comment by klues** [comment]
"Search provider" is the background service used to fetch the images:
* ARASAAC: the official ARASAAC API: https://arasaac.org/developers/api
* OPENSYMBOLS: https://www.opensymbols.org/

Opensymbols also contains ARASAAC images (among others), but in their API they've added the author "Sergio Paulo", which is the designer of the ARASAAC images.

**Comment by ms-mialingvo** [comment]
So, then its an issue of the ARASAAC API, @arasaac-dga , that the API doesn't include all images that are created by Sergio. Like for example in the screenshot, the october calender with the pumpkin does not appear when filtering for ARASAAC images but it's created by Sergio in the ARASAAC style.

...except if not all images from Sergio are made for ARASAAC / made as the designer of ARASAAC, because he's freelancing separatedly.

**Comment by arasaac-dga** [comment]
The Creative Commons licence under Arasaac pictograms are distributed allow derivated works. So under this condition some Global Symbols users have crreated other catalogs with customiced pictograms based on Arasaac. But they are not official and recommended by us pictograms but perhaps valud for them. The Globalsymbols Api mixes all the catalogs being crazy.  So we don't recommend in any case use Globalsymbols search. Official pictograms only using our API. Sergio is the creator of the  graphic style and that's why appear as author in our API

**Comment by ms-mialingvo** [comment]
So, even though he's explicitedly named as the author it's not him who has created these graphics then?

<img width="177" alt="Image" src="https://github.com/user-attachments/assets/072c9c9b-b805-4420-a369-4cba68e8cc80" />

If yes, then the crediting is a bit confusing but then I can understand why it doesn't appear in the ARASAAC filter.

**Comment by arasaac-dga** [comment]
No. Sergio is not the author of the derivated work. The problem is from Globalsymbols that not specify that is a derivated work from Arasaac symbols. We commented them that is not clear in those catalogs that are derivated works from Arasaac symbols. Other problem that they have is that in some cataligs not all the pictograms are derivated works. Some pictos are originals without modifications, other derivated works and others new designs. A chaos. They would have to indentify individually each pictogram and if it is original, derivated or New.

**Comment by klues** [comment]
> If yes, then the crediting is a bit confusing but then I can understand why it doesn't appear in the ARASAAC filter.

It's not a filter, but the general option where all the images come from - either the ARASAAC API or the opensymbols API.
And I'm just displaying the data from the opensymbols API, which states that the author is Sergio Palao, so the problem is the non-accurate data from the opensymbols API.

> The problem is from Globalsymbols that not specify that is a derivated work from Arasaac symbols.

It's opensymbols, not Globalsymbols (I know, it's confusing). Globalsymbols also have an API and a collection of open pictograms, but I haven't integrated it yet, but there is this issue: https://github.com/asterics/AsTeRICS-Grid/issues/79
Globalsymbols is also the partner, where the plan is to do the communication board sharing functionality with their platform "BoardBuilder".

**Comment by arasaac-dga** [comment]
Investigating the "October" case a little more, as you can see here it belongs to a different catalogue than ARASAAC https://www.opensymbols.org/repositories/coughdrop_symbols in which there are only 22 pictograms referring to calendar and days of the week. The calendar ones are derived works of ARASAAC and those of the days of the week are not from ARASAAC. As you can see when entering the pictograph file https://www.opensymbols.org/symbols/coughdrop_symbols/october-ee801a2d?id=43476 the author is Sergio Palao which is the information that appears when searching in Asterics. The problem is the one we were also commenting on for Global Symbols (sorry for the confusion) that it is not identified that it is a derivative work that comes from ARASAAC and Sergio is identified as the author who is the author of the ARASAAC graphic line but is not the author of that Derivative Work.

Furthermore, in OpenSymbols, not even the ARASAAC website is properly referenced, given that it was the first one we had many years ago. It's a bit of a mess.

If both OpenSymbols and GlobalSymbols really wanted to do things right, each pictogram would have to be individually identified and specified if it is an original ARASAAC pictogram and, in that case, mention the ARASAAC origin, CC BY-NC-SA license and author Sergio Palao, and if it is a derivative work based on ARASAAC, specify it clearly with a text similar to: "Derived work from ARASAAC (https://arasaac.org) under CC BY-NC-SA" or similar.

And that correct information is what the API should return.

**Comment by klues** [comment]
thanks for your investigation and comment, I've copied it to an issue at globalsymbols: https://github.com/open-aac/opensymbols/issues/31
